runtimes/pythonrt/pythonrt.go

```diff
@@ -280,22 +311,34 @@ func (py *pySvc) Run(
 	py.comm = NewComm(conn)
 
 	// FIXME (ENG-577) We might get activity calls before module is loaded if there are module level function calls.
+	var mod ModuleMessage
+	for {
+		msg, err := py.comm.Recv()
+		if err != nil {
+			py.log.Error("initial message from python", zap.Error(err))
+			return nil, err
+		}
 
-	msg, err := py.comm.Recv()
-	if err != nil {
-		py.log.Error("initial message from python", zap.Error(err))
-		return nil, err
-	}
+		if msg.Type == "" {
+			py.log.Error("python can't load module", zap.String("module", mainPath))
+			return nil, fmt.Errorf("python can't load %q", mainPath)
```

Comment:
why do you log and turn the exact same error ?

Reply:
Not sure how `ak` logs it, helps me with debugging.